write-combining/write-combining.cpp

```diff
@@ -1,10 +1,11 @@
 #include <iostream>
+#include <iomanip>
 #include <chrono>
 #include <thread>
 #include <cassert>
 #include <vector>
 #include <cstring>
-#include <xmmintrin.h>
+#include <immintrin.h>
```

Comment:
It is recommended to use `immintrin.h` rather than the specific files like `xmmintrin`.